plugins/plugin_template/__gui_project__/CMakeLists.txt

```diff
@@ -0,0 +1,69 @@
+project(__project__)
+set(PROJECT_VERSION 1.0)
+set(PROJECT_DESCRIPTION "__description__")
+cmake_minimum_required(VERSION 2.8.11)
+include(FindPkgConfig)
+
+find_package(mars)
+mars_defaults()
+define_module_info()
+
+find_package(Qt5Widgets REQUIRED)
```

Comment:
Please use the same method for finding packages than the other mars compoents to make sure they are consistent even qt5 is not used. (which is currently the case in most setups)

See: https://github.com/rock-simulation/mars/blob/master/gui/CMakeLists.txt